R/row-redux.R

```diff
@@ -156,3 +156,12 @@ col_concat <- function(data, sep=""){
   apply(data, 1, paste0, collapse=sep)
 }
 attr(col_concat, "call") <- "col_concat"
+
+#' @export
+is_col_duplicated <- function(data, allow.na = FALSE) {
+  if(!(any(class(data) %in% c("matrix", "data.frame"))))
+    stop("\"data\" must be a data.frame (or matrix)", call.=FALSE)
```

Comment:
Yeah sure, but it will not follow the styleguide of the repository. There are some plans to merge or fork with the main one so I think we should leave it as is. What do you think?

Reply:
Ok, let's leave it as it is then